docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst

```diff
@@ -0,0 +1,190 @@
+=========
+Audit Log
+=========
+
+The Audit Log is a feature that allows you to view and filter different events within a user's account.
+It is located in the administrative panel
+
+.. image:: pic_auditLog/auditLogAdmin.png
+   :width: 600
+   :align: center
+
+Manage audit log
+================
+
+1. When you open the audit log, you will see all the events that occurred within the account on the current day for the current user.
```

Comment:
"for the current user." is redundant information as it's not true and records will be written for all users who has access to the account

Reply:
done